cli/command/trust/signer_remove.go

```diff
@@ -120,16 +122,21 @@ func removeSingleSigner(cli command.Cli, repoName, signerName string, forceYes b
 
 		if !removeSigner {
 			fmt.Fprintf(cli.Out(), "\nAborting action.\n")
-			return nil
+			return false, nil
 		}
 	} else if err != nil {
-		return err
+		return false, err
 	}
 	if err = notaryRepo.RemoveDelegationKeys(releasesRoleTUFName, role.KeyIDs); err != nil {
-		return err
+		return false, err
 	}
 	if err = notaryRepo.RemoveDelegationRole(signerDelegation); err != nil {
-		return err
+		return false, err
+	}
+
+	if err = notaryRepo.Publish(); err != nil {
+		return false, err
 	}
-	return notaryRepo.Publish()
+
+	return true, nil
```

Comment:
Wouldn't it be simpler to move the logged success message before this return?
```
    fmt.Fprintf(cli.Out(), "Successfully removed %s from %s\n\n", signerName, repoName)
    return nil
```
?

Reply:
I'd say that I prefer to have it outside as it makes it more maintainable in the case where there are different codepaths that lead to a successful removal in this function and simply have this one return whether or not a removal happened.

Reply:
OK but for now there is only one code path which does that, this would minimize the changes required and there is already output logged in a similar way in the function e.g.
```
fmt.Fprintf(cli.Out(), "\nAborting action.\n")
```
Anyway it's just my personal view, obviously :)

Reply:
Makes sense! fixed.